src/docs/development/add-to-app/android/add-flutter-screen.md

```diff
@@ -122,78 +47,79 @@ myButton.addOnClickListener(new OnClickListener() {
   @Override
   public void onClick(View v) {
     startActivity(
-      FlutterActivity.createBuilder().build(currentActivity)
+      FlutterActivity.createDefaultIntent(currentActivity)
     );
   }
 });
 ```
 
 The above example assumes that your Dart entrypoint is called `main()`, and your
-initial Flutter route is '/'. If you'd like to change either property, use the
-`IntentBuilder` to provide your desired properties:
+initial Flutter route is '/'. The Dart entrypoint cannot be changed via `Intent`,
+but the initial route can be changed via `Intent`. The following example
+demonstrates how to launch a `FlutterActivity` that initially renders a custom
+route in Flutter.
 
 ```java
 myButton.addOnClickListener(new OnClickListener() {
   @Override
   public void onClick(View v) {
     startActivity(
       FlutterActivity
-        .createBuilder()
-        .dartEntrypoint("mySpecialEntrypoint")
+        .withNewEngine()
         .initialRoute("/my_route")
         .build(currentActivity)
       );
   }
 });
 ```
 
-Replace `"mySpecialEntrypoint"` with the name of your desired Dart entrypoint,
```

Comment:
Also, replace that ending comma with a period.  ;)

Reply:
This line was deleted. Is there an issue with the new line that was added?